plugins/recruitee/src/dataSources.ts

```diff
@@ -0,0 +1,311 @@
+        { id: "closed_at", name: "Closed At", type: "string" },
+        { id: "priority", name: "Priority", type: "string" },
+        {
+            id: "recruiter_id",
+            name: "Recruiter",
+            type: "string",
+        },
+        { id: "hybrid", name: "hybrid", type: "boolean" },
+        { id: "wysiwyg_editor", name: "Wysiwyg Editor", type: "string" },
+        { id: "created_at", name: "Created At", type: "string" },
+        { id: "updated_at", name: "Updated At", type: "string" },
+        { id: "guid", name: "Guid", type: "string" },
+        { id: "example", name: "Example", type: "boolean" },
+        { id: "followed", name: "Followed", type: "boolean" },
+        {
+            id: "location",
+            name: "Location",
+            type: "string",
+        },
+        {
+            id: "location_ids",
+            name: "Locations",
+            type: "multiCollectionReference",
+            collectionId: "",
+            dataSourceId: "Locations",
+        },
+    ]
+)
+const departmentsSchema = v.array(v.object({ departments: v.array(DepartmentSchema) }))
+const departmentsDataSource = createDataSource(
+    {
+        name: "Departments",
+        fetch: async (token: string, companyId: string) => {
+            const url = `https://api.recruitee.com/c/${companyId}/departments`
+            const data = v.safeParse(departmentsSchema, await fetchRecruiteeData(url, token))
+            if (!data.success) {
+                console.log("Error parsing Recruitee data:", data.issues)
+                throw new Error("Error parsing Recruitee data")
+            }
+            return data.output.flatMap(page => page.departments)
+        },
+    },
+    [
+        { id: "id", name: "ID", type: "string", canBeUsedAsSlug: true },
+        { id: "name", name: "Name", type: "string", canBeUsedAsSlug: true },
+        { id: "status", name: "Status", type: "string" },
+        { id: "offers_count", name: "Offers Count", type: "number" },
+        { id: "talent_pools_count", name: "Talent Pools Count", type: "number" },
+    ]
+)
+
+const candidatesSchema = v.array(v.object({ candidates: v.array(CandidateSchema) }))
+const candidatesDataSource = createDataSource(
+    {
+        name: "Candidates",
+        fetch: async (token: string, companyId: string) => {
+            const url = `https://api.recruitee.com/c/${companyId}/candidates`
+            const data = v.safeParse(candidatesSchema, await fetchRecruiteeData(url, token))
+            if (!data.success) {
+                console.log("Error parsing Recruitee data:", data.issues)
+                throw new Error("Error parsing Recruitee data")
+            }
+            return data.output.flatMap(page => page.candidates)
+        },
+    },
+    [
+        { id: "id", name: "ID", type: "string", canBeUsedAsSlug: true },
+        { id: "initials", name: "Initials", type: "string" },
+        { id: "name", name: "Name", type: "string", canBeUsedAsSlug: true },
```

Comment:
```suggestion
        { id: "name", name: "Name", type: "string", canBeUsedAsSlug: true },
        { id: "initials", name: "Initials", type: "string" },
        { id: "id", name: "ID", type: "string", canBeUsedAsSlug: true },
```

Make name first in the fields list to improve the candidates CMS view.
<img width="751" height="508" alt="image" src="https://github.com/user-attachments/assets/a5d35730-078e-450f-8c84-ad3679441c55" />